src/FsAutoComplete/CodeFixes/AddMissingXmlDocumentation.fs

```diff
@@ -14,21 +14,20 @@ let title = "Add missing XML documentation"
 
 let private tryGetExistingXmlDoc (pos: FSharp.Compiler.Text.Position) (xmlDoc: PreXmlDoc) =
   let tryGetSummaryIfContainsPos (xd: PreXmlDoc) =
-    let d = xd.ToXmlDoc(false, None)
-
-    if rangeContainsPos d.Range pos then
+    if rangeContainsPos xd.Range pos then
+      let d = xd.ToXmlDoc(false, None)
       if Array.isEmpty d.UnprocessedLines then
         None
       elif d.UnprocessedLines |> Array.exists (fun s -> s.Contains("<summary>")) then
-        Some(d.UnprocessedLines, d.Range)
+        Some(d.UnprocessedLines, xd.Range)
```

Comment:
use xd for consistency

Reply:
isn't `xd.Range` a mistake, `xd` is not defined.

The above variable is still defined as `d`.

Reply:
See line 17, `xd` is the `PreXmlDoc` passed to the function.